utils/releaser/src/ReleaseWorker/StageWorkerInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Shopsys\Releaser\ReleaseWorker;
+
+use PharIo\Version\Version;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\StageAwareInterface;
+
+interface StageWorkerInterface extends StageAwareInterface
+{
+    /**
+     * 1 line description of what this worker does, in a commanding form! e.g.:
+     * - "Add new tag"
+     * - "Dump new items to CHANGELOG.md"
+     * - "Run coding standards"
+     *
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string;
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
+     */
+    public function work(Version $version, string $initialBranchName = 'master'): void;
```

Comment:
initialBrancheName is required, so default value may not be here ?

Reply:
Sadly interface says, that this method has only one argument, so in order to add another argument, it has to have default value set

Reply:
oki, in first view looks like new interface instead of ReleaseWorkerInterface